content/en/continuous_integration/pipelines/jenkins.md

```diff
@@ -360,7 +290,78 @@ Re/Initialize Datadog-Plugin Agent Http Client
 TRACE -> http://<HOST>:<TRACE_PORT>/v0.3/traces
 {{< /code-block >}}
 
-## Correlate infrastructure metrics
+## Install the Datadog Agent (optional)
+
+The Datadog Jenkins plugin can either report metrics through the Datadog Agent or directly to Datadog if an API key is provided. If you don't have a Datadog Agent running on the Jenkins controller instance, Datadog recommends installing it first by following the [Agent installation instructions][14].
```

Comment:
Clicking on the `agent-based` link at the top of the page will take the reader directly to this section, skipping the plugin installation instructions. Besides, the plugin installation instructions now say "agentless" in the title, so the users might think that the plugin is not needed for agent-based setup.

Perhaps we could make it more explicit here, that the plugin is required regardless of whether the agentless or the agent-based mode is used

Reply:
```suggestion
The Datadog Jenkins plugin can either report metrics through the Datadog Agent or directly to Datadog if an API key is provided. If you don't have a Datadog Agent running on the Jenkins controller instance, Datadog recommends installing it first by following the [Agent installation instructions][14]. Whether you choose to use Agentless mode or the Agent-based mode, you are required to use the plugin. 
```